city/crud.py

```diff
@@ -0,0 +1,51 @@
+from sqlalchemy.orm import Session
+
+from city import models, schemas
+
+
+def create_city(db: Session, city: schemas.CityCreateUpdate) -> models.City:
```

Comment:
Consider adding error handling for database operations. For example, if the commit fails, it would be beneficial to catch exceptions and handle them appropriately to avoid application crashes.